yt_dlp/extractor/xvideos.py

```diff
@@ -223,3 +230,114 @@ class XVideosQuickiesIE(InfoExtractor):
     def _real_extract(self, url):
         domain, id_ = self._match_valid_url(url).group('domain', 'id')
         return self.url_result(f'https://{domain}/video{"" if id_.isdecimal() else "."}{id_}/_', XVideosIE, id_)
+
+
+class XVideosUserIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:.+?\.)?xvideos\.(?:com|es)/
+        (?P<page_path>(?:channels|amateur-channels|model-channels|pornstar-channels|profiles)/
+        (?P<id>[^/?#&]+))((?:(?!\#quickies).)+)?$'''
```

Comment:
```suggestion
        (?P<id>[^/?#&]+))(?!\#quickies$)'''
```
maybe?

Reply:
Well, no. `https://xvideos.com/profiles/foo#quickies/bar` will be matched as `('profiles/fo', 'fo')` .

Reply:
```suggestion
        (?P<id>[^/?#&]+))(?:(?:(?!\#quickies).)+)?$'''
```
then at least do this, this capturing group seems to be unused. You can also override the function `_VALID_URL` when you're trying to avoid two extractors to match the same url

Reply:
> You can also override the function `_VALID_URL` when you're trying to avoid two extractors to match the same url

Thanks, this is the first time I know about `_VALID_URL(cls)`. I'll try it if needed.

Reply:
> You can also override the function `_VALID_URL` when you're trying to avoid two extractors to match the same url

Hmm, how? The `_VALID_URL(cls)` function returns a regex pattern. How do I *avoid two extractors to match the same url* by it?

Reply:
I think they mean `suitable()`, e.g.:

https://github.com/yt-dlp/yt-dlp/blob/4a9bc8c3630378bc29f0266126b503f6190c0430/yt_dlp/extractor/twitch.py#L758-L764

but using negative lookahead in the regex pattern to like you are currently doing is acceptable too IMO